src/prog_models/prognostics_model.py

```diff
@@ -1344,7 +1339,7 @@ def generate_surrogate(self, load_functions, method = 'dmd', **kwargs):
         from prog_models.data_models import SURROGATE_METHOD_LOOKUP
 
         if method not in SURROGATE_METHOD_LOOKUP.keys():
-            raise ProgModelInputException("Method {} not supported. Supported methods: {}".format(method, SURROGATE_METHOD_LOOKUP.keys()))
+            raise ValueError("Method {} not supported. Supported methods: {}".format(method, SURROGATE_METHOD_LOOKUP.keys()))
```

Comment:
NotImplementedError?

Reply:
NotImplemented is for when there's a functionality that is not implemented (e.g., next_state function in subclass). In this case the value of method is incorrect (because it's not one of the accepted values, i.e., those from SURROGATE_METHOD_LOOKUP). That's why we choose ValueError here

Reply:
That makes perfect sense